### ISSUE TITLE
Failed to install the package error

### ISSUE THREAD

**Issue by moatazeldebsy** [kobiton/bitrise-step-kobiton-execute-test#2]
Hi team, I was trying to use the Bitrise Step in my workflow and I got this error

```
+------------------------------------------------------------------------------+

| (2) kobiton-execute-test@1                                                   |
+------------------------------------------------------------------------------+
| id: kobiton-execute-test                                                     |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2022-03-01T16:55:09Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
# github.com/kobiton/bitrise-step-kobiton-execute-test
src/github.com/kobiton/bitrise-step-kobiton-execute-test/main.go:57:34: time.Now().UnixMilli undefined (type time.Time has no field or method UnixMilli)
src/github.com/kobiton/bitrise-step-kobiton-execute-test/main.go:68:33: time.Now().UnixMilli undefined (type time.Time has no field or method UnixMilli)
ERRO[16:55:09] Step (kobiton-execute-test@1) failed: Failed to prepare the step for execution through the required toolkit (go), error: Failed to install package, error: command `/usr/local/bin/go "build" "-o" "/Users/vagrant/.bitrise/toolkits/go/cache/https___github.com_bitrise-io_bitrise-steplib.git-kobiton-execute-test-1" "github.com/kobiton/bitrise-step-kobiton-execute-test"` failed: exit status 2
```

**Comment by Nhattd97** [comment]
Hi @moatazeldebsy , this issue comes from Bitrise which downgrades the Go version from 1.17.x to 1.16.x. So it breaks some functions that are just available at 1.17.x.
Let me contact with Birise team for more details.

**Comment by Nhattd97** [comment]
[Fix version: `1.0.1`]
This issue should be fixed once this [PR](https://github.com/bitrise-io/bitrise-steplib/pull/3425) is merged by the Bitrise team.

**Comment by moatazeldebsy** [comment]
Thank you @Nhattd97 will try it
